internal/routes.go

```diff
@@ -140,6 +142,16 @@ func fetchMovie(id string, movieCh chan Movie) {
 
 	defer res.Body.Close()
 
+	if res.StatusCode == 429 {
+		fmt.Println(res.Status, id, "- Will retry in 1 second")
+		time.Sleep(1)
+		fetchMovie(id, movieCh)
+	}
+
+	if res.StatusCode != 200 {
```

Comment:
same as above

Reply:
Done - Did both for TV and Person as well

Reply:
Beautiful 🤘